plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1356,6 +1356,22 @@ public void testPartitionColumnNameConflict()
         }
     }
 
+    @Test
+    public void testAlterPartitionColumn()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_alter_partition", "(_time timestamp, ts timestamp) WITH (partitioning = ARRAY['day(_time)'])")) {
+            assertUpdate("ALTER TABLE " + table.getName() + " SET PROPERTIES partitioning = ARRAY['day(_time)','ts']");
```

Comment:
Could you verify `SHOW CREATE TABLE` result as well?

Reply:
added the check to verify `SHOW CREATE TABLE` result